h/views/activity.py

```diff
@@ -177,6 +191,10 @@ def user_annotation_count(aggregation, userid):
                 'The group “{name}” has not made any annotations yet.').format(
                     name=Markup.escape(self.group.name)))
 
+        result['logged_in_user_can_leave_group'] = False
+        if self.request.user in self.group.members:
+            result['logged_in_user_can_leave_group'] = True
```

Comment:
This could just be `result['logged_in_user_can_leave_group'] = self.request.user in self.group.members`. I'd probably have called it `show_leave_button` since that's what it does, too.